engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
```

Comment:
as i said in the other comment - failed subscriptions might be important. Braintree is probably much better at displaying everything else.